roles/matrix-postgres/tasks/setup_postgres.yml

```diff
@@ -73,10 +73,10 @@
     mode: 0750
   when: matrix_postgres_enabled|bool
 
-- name: Ensure matrix-make-user-admin script created
+- name: Ensure matrix-change-user-admin-status script created
   template:
-    src: "{{ role_path }}/templates/usr-local-bin/matrix-make-user-admin.j2"
-    dest: "/usr/local/bin/matrix-make-user-admin"
+    src: "{{ role_path }}/templates/usr-local-bin/matrix-change-user-admin-status.j2"
+    dest: "/usr/local/bin/matrix-change-user-admin-status"
     mode: 0750
   when: matrix_postgres_enabled|bool
```

Comment:
It's probably good to also add a cleanup task for the old file as well. Something like:

```yaml
- name: (Migration) Ensure old matrix-make-user-admin script deleted
  file:
    path: /usr/local/bin/matrix-make-user-admin
    state: absent
  when: matrix_postgres_enabled|bool
```